src/Maestro/maestro-angular/src/app/page/subscriptions-table/subscriptions-table.component.html

```diff
@@ -4,7 +4,12 @@
     <th>Source Channel</th>
     <th>Target Branch</th>
     <th>Status</th>
+    <th>Update Frequency</th>
+    <ng-container *ngIf="openDetails">
+      <th>Type</th>
+    </ng-container>
   </thead>
+  <button (click)="openDetails=!openDetails">Show/Hide Additional Details</button>
```

Comment:
This button confuses me because of where it's placed and what it's labeled.  I can't look at the page without feeling like I have to push this button.  Is it possible to move it up into the "Subscriptions" bar and call it something like "Load Version.Details.xml details"?  

I wonder if, instead, we had a dropdown that defaulted to not showing anything, but allowed you to select a branch, then it would load the Version.Details.xml file from that branch (to go along with my earlier comment)?

@mmitche?